README.md

```diff
@@ -1,51 +1,98 @@
 # Kik Bot API #
-Use this library to develop bots for [Kik Messenger](https://www.kik.com) that are essentially automated humans.
+The Kik Bot API is a Python library developed to automate interactions on [Kik Messenger](https://www.kik.com) that are essentially automated humans.
 
-It basically lets you do the same things as the official Kik app by pretending to be a real smartphone client; It communicates with Kik's servers at `talk1110an.kik.com:5223` over a modified version of the [XMPP](https://xmpp.org/about/technology-overview.html) protocol.
+It's essentially a way to create bots that behave like humans on the platform. This library enables your bot to interact with the official Kik app by emulating a real smartphone client. It communicates with Kik's servers at `talk1110an.kik.com:5223` over a modified version of the [XMPP](https://xmpp.org/about/technology-overview.html) protocol.
+
+This library is ideal for developers, hobbyists, and businesses who want to build automated bots to interact with users, groups, and other bots on Kik.
+
+We do not endorse the use of this library for spamming or other malicious purposes. Please use this library responsibly.
+
+## Features ##
```

Comment:
Do you think it's better the have the Features section appear before the usage example or after?

Reply:
Not sure, most people describe the project before showing how to use it tho

Reply:
I think it's better the have the usage example first, to get an idea of what this library looks like, and then list the possible additional features